tbx/static_src/javascript/components/desktop-sub-menu.test.js

```diff
@@ -0,0 +1,129 @@
+import DesktopSubMenu from './desktop-sub-menu';
+
+describe('DesktopSubMenu', () => {
+    beforeEach(() => {
+        document.body.innerHTML = `
+            <div data-desktop-menu="">
+                <nav aria-label="Primary">
+                    <ul class="primary-nav" data-primary-nav="">
+                        <li class="primary-nav__item primary-nav__item--is-parent" data-has-subnav="">
```

Comment:
Question: Do we need to strip out all these classes as well?

Reply:
ah very good spot

Reply:
Actually I think I'll leave it for now, as we don't know what the markup will be. I'll just add a todo comment to update it when the navigation is implemented.